lib/ansible/utils/import_module.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+# HACK: keep Python 2.6 controller tests happy in CI until they're properly split
+try:
+    from importlib import import_module
+except ImportError:
+    # importlib.import_module returns the tail
+    # whereas __import__ returns the head
+    # compat to work like importlib.import_module
+    def import_module(name):
+        module = __import__(name)
+        for part in name.split('.')[1:]:
+            module = getattr(module, part)
+        return module
```

Comment:
Is the following perhaps a better implementation?

https://github.com/ansible/ansible/blob/4e8b240b8b74db2453851f12ec82c7f782289187/test/lib/ansible_test/_data/sanity/import/importer.py#L30-L32

Reply:
Hm, I think so, I didn't think about doing that

Reply:
It turns out we already have this:

https://github.com/ansible/ansible/blob/4e8b240b8b74db2453851f12ec82c7f782289187/lib/ansible/module_utils/six/__init__.py#L89-L92

Reply:
Unfortunately it is apparently private, as it is undocumented, as @sivel noticed.